Lib/selectors.py

```diff
@@ -580,16 +581,31 @@ def close(self):
             super().close()
 
 
+def _can_use(method):
+    """Check if we can use the selector depending upon the
+    operating system. """
+    # Implementation based upon https://github.com/sethmlarson/selectors2/blob/master/selectors2.py
+    try:
+        if method == 'poll':
+            poll_obj = select.poll()
+            poll_obj.poll(0)
```

Comment:
Don’t you also have to close this?

Reply:
Pardon me, I didn't understand this comment. There is no close method associated with this object.

Reply:
Hm, you're right. I forgot how this works.

However I have another thought. Skimming the C code in Modules/selectmodule.c I came across the `HAVE_BROKEN_POLL` define, and the `select_have_broken_poll()` function. I wonder if it isn't better if we arranged for the platform you're testing on (which I have never heard of) to either set `HAVE_BROKEN_POLL` or cause that function to return 1?